src/table.ts

```diff
@@ -940,13 +963,21 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
             numRequestsMade <= maxRetries &&
             RETRYABLE_STATUS_CODES.has(error.code)
           ) {
-            makeNewRequest();
+            const backOffSettings =
+              options.gaxOptions?.retry?.backoffSettings ||
+              DEFAULT_BACKOFF_SETTINGS;
+            const nextRetryDelay = getNextDelay(
+              numRequestsMade,
```

Comment:
should this be numConsecutiveErrors? In other words should backoff be rest when receiving a successful result? Maybe add a TODO to figure this out